quickstart.rb

```diff
@@ -1,7 +1,7 @@
 require 'fileutils'
 
 class Quickstart < Formula
-  v = "v1.2.0"
+  v = "v1.3.0"
```

Comment:
Quickstart had a dot release today (1.3.1), should we use that version instead of 1.3.0? 
(possibly the same question for the other plugins)

Reply:
good catch - i had the right SHAs